analytics/src/analytics/integrations/etldb/main.py

```diff
@@ -18,7 +18,7 @@
 VERBOSE = False
 
 
-def initialize_database() -> None:
+def db_migrate() -> None:
```

Comment:
nit: please change this to `migrate_database`

Reply:
https://github.com/HHS/simpler-grants-gov/pull/2978/commits/84ec4d70482be42b21e9717ccc5d58bc80b59346 ✅